src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
```

Comment:
The class name 'Rectangle' is appropriate as it is a noun. However, ensure that any interfaces it implements have names that are not verbs to align with best practices.